crates/fluvio-cli/src/client/home/status.rs

```diff
@@ -27,12 +28,26 @@ impl StatusOpt {
             .filter_map(|item| {
                 match item.spec.remote_type {
                     RemoteType::Core(core) => {
+                        let now = SystemTime::now()
+                            .duration_since(SystemTime::UNIX_EPOCH)
+                            .unwrap_or_default()
```

Comment:
add unit test and also seems duplicated twice

Reply:
Right! About "duplicated twice" is about the home/status.rs and remote/list.rs code?

Reply:
I added a new method `last_seen` in RemoteStatus with tests